core/thread.py

```diff
@@ -277,17 +358,21 @@ def _close_after(self, closer, silent, delete_channel, message):
                    from_mod=False, note=False, anonymous=False):
         if self.close_task is not None:
             # cancel closing if a thread message is sent.
-            await self.cancel_closure()
-            await self.channel.send(embed=discord.Embed(
-                color=discord.Color.red(),
-                description='Scheduled close has been cancelled.'
-            ))
+            tasks = asyncio.gather(
+                self.cancel_closure(),
+                self.channel.send(embed=discord.Embed(
+                    color=discord.Color.red(),
+                    description='Scheduled close has been cancelled.'
+                    )))
+            self.bot.loop.create_task(tasks)
 
         if not from_mod and not note:
-            await self.bot.api.append_log(message, self.channel.id)
+            self.bot.loop.create_task(
+                self.bot.api.append_log(message, self.channel.id)
+            )
 
         if not self.ready:
-            await self.wait_until_ready()
+            self.wait_until_ready()
```

Comment:
`wait_until_ready` is a coroutine, but never awaited? https://github.com/kyb3r/modmail/pull/172/files#diff-02108bdf20ae302859bd16f5e0c1dd1bR41